meeting-notes/2018/2018-06-11--js-ipfs-dev-team-weekly.md

```diff
@@ -0,0 +1,119 @@
+# ⒿⓈ⚡️  js-ipfs Core Dev Team Weekly 2018-06-11
+
+- **Lead:** Jacob Heun (@jacobheun)
+- **Notetaker:** Diogo Silva (@fsdiogo)
+- **Attendees:**
+  - @achingbrain
+  - @fsdiogo
+  - @vasco-santos
+  - @hugomrdias
+  - @gar
+- **Recording:** _add link to recording once it's online_
```

Comment:
@jacobheun @fsdiogo where is the recording for this call?

Reply:
Check this https://github.com/ipfs/pm/pull/643#issuecomment-398060247.

Reply:
@diasdavid the recording should be in your cloud recordings, https://protocol.zoom.us/recording. Sorry about that, I selected cloud recording and incorrectly assumed I would have access to it, but it only looks like alternate hosts can start/stop the recording. I will use the local recording in the future.